scr/index.html

```diff
@@ -49,6 +49,150 @@ <h1 class="headline-h1 h1Color">I’m Aisha.  <br>
+</section>
+</section>
+<!--------------------Works Container3-------------------->
+<section class="work-container1">
+
+  <img src="../Images/3.png" alt="">
+<div class="work">
+<h1>Facebook 360</h1>
+<div class="facebook"> <p>FACEBOOK</p>
+<ul>
+  <li>Full Stack Dev</li>
+  <li>2015</li>
+</ul>
+</div>
+
+<p class="work-p">Exploring the future of media in Facebook's first Virtual Reality app; a place to 
+discover and enjoy 360 photos and videos on Gear VR.</p>
+<ul class="work-list">
+  <li><a href=""> html</a></li>
+  <li class="desktop_link"><a href=""> Ruby on rails</a></li>
+  <li><a href=""> css</a></li>
+  <li><a href=""> javaScript</a></li>
+</ul>
+<button class="btn">See project</button>
+</div>
+</section>
+</section>
+<!--------------------Works Container4-------------------->
+<section class="work-container2" >
+
+  <img src="../Images/4.png" alt="">
+<div class="work">
+<h1>Uber Navigation</h1>
+<div class="facebook"> <p>Uber</p>
+<ul>
+  <li>Lead Developer</li>
+  <li>2018</li>
+</ul>
+</div>
+
+<p class="work-p">A smart assistant to make driving more safe,
+ efficient, and fun by unlocking your most expensive computer: your car.</p>
+ <ul class="work-list">
+  <li><a href=""> html</a></li>
+  <li class="desktop_link"><a href=""> Ruby on rails</a></li>
+  <li><a href=""> css</a></li>
+  <li><a href=""> javaScript</a></li>
+</ul>
+<button class="btn">See project</button>
+</div>
+</section>
+<!--------------------About Me-------------------->
+<section class="about-container" id="About-container">
+<div class="about-r">
+  <h1 class="about-text h1Color">About Myself</h1>
+  <p>Hello I’m a software developer! I can help you build a product , feature or website Look through 
+    some of my work and experience! If you like what you see and have a project you need coded, don’t hesitate to contact me.</p>
+      <p class="about-connect main_color">LET’S CONNECT</p>
+      <div class="about-icon">
+        <ul class="social-media-icons"> 
+          <li><a href="#"><img class="icon" src="../Images/twitter.svg" alt=""></a></li>
+          <li><a href="#"><img class="icon" src="../Images/Linkedin icon.svg" alt=""></a></li>
+          <li><a href="#"><img class="icon" src="../Images/medium.svg" alt=""></a></li>
+          <li><a href="#"><img class="icon" src="../Images/github.svg" alt=""></a></li>
+          <li><a href="#"><img class="icon" src="../Images/angellist.svg" alt=""></a></li>
+        </ul>
+     </div>
+  <button class="btn">See project</button>
+</div>
+<div class="about-l">
+<ul class="aboutme-list">
+  <li> <div class="Languages"><h2>Languages</h2> <i class='bx bx-chevron-down'></i></div> 
+   <div class="lang">
+   <div class="lang-bord">
+    <img src="../Images/js.png" alt="">
+    <h1>JavaScript</h1>
+
+   </div>
+   <div class="lang-bord">
+    <img src="../Images/html.png" alt="">
+    <h1>HTML</h1>
+
+   </div>
+   <div class="lang-bord">
+   <img src="../Images/css.png" alt="">
+<h1>CSS</h1>
+</div>
+</div>

```

Comment:
Great job on your languages section! I have a few suggestions **that can help make it closer to the provided [Figma template](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0-1&t=X3BgW0yrAVqXHazX-0) and follow HTML best practices**:

- [ ] Instead of using different `div` elements for each of your languages, consider using another `ul` list with `li` tags for each language. **This approach can help you create a more semantic and organized structure for your content.** 🙋‍♂️

- [ ] Check the `font-size` for your language titles, which should be `15px` instead of `12px`. Additionally, make sure the `font-weight` is set to **500** to match the [Figma template](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0-1&t=X3BgW0yrAVqXHazX-0). 👍

- [ ] As one of my previous suggestions, it's best to avoid using multiple `h1` tags on a single page. Consider using other heading tags like `h2, h3, etc`. to organize your content and improve accessibility. 😉